web/app/components/document/sidebar/related-resources/list.hbs

```diff
@@ -6,6 +6,8 @@
       </div>
     {{else}}
       <ul class="related-resources-list" ...attributes>
+        {{! This is being looped through twice for some reason... }}
+        {{! On immediate save, the order is wrong - maybe an array push thing? }}
```

Comment:
Should we follow up on this (/add a TODO)?

Reply:
This was actually resolved and I forgot to remove the comment. Thanks for catching.